dealership/contracts.py

```diff
@@ -1,12 +1,61 @@
 class Contract(object):
-    pass
+
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+
+    def get_discount(self):
+        if self.customer.is_employee():
+            discount = .1
+        else:
+            discount = 0
+        return discount
 
 
 class BuyContract(Contract):
+
+    interest = {
+        'Car': 1.07,
+        'Motorcycle': 1.03,
+        'Truck': 1.11
+     }
+
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        Contract.__init__(self, vehicle, customer)
```

Comment:
You should use `super` rather than calling the parent method directly:
`super(BuyContract, self).__init__(self, vehicle, customer)`

Reply:
Thanks. I initially learned it by calling the parent method directly but that has probably been replaced by using `super`.